Test/SeqCoverageTest.cs

```diff
@@ -1,11 +1,15 @@
-﻿using EngineLayer;
+﻿using System;
+using EngineLayer;
 using NUnit.Framework;
 using Proteomics;
 using Proteomics.Fragmentation;
 using Proteomics.ProteolyticDigestion;
 using System.Collections.Generic;
 using System.Linq;
 using MassSpectrometry;
+using FlashLFQ;
+using static iText.IO.Image.Jpeg2000ImageData;
```

Comment:
what the heck is that? looks like a virus

Reply:
Yikes... no idea but I removed it.